components/molecules/HighlightCard/highlight-card.tsx

```diff
@@ -108,51 +111,55 @@ const HighlightCard: React.FC<HighlightCardProps> = ({
         </div>
 
         {/* Main Information */}
-        <div className="flex flex-col w-full px-6 pb-5 mt-2">
-          {/* Main Number */}
-          <div className="flex flex-col items-center">
-            {/* Percentage */}
-            <div className="text-4xl">
-              {percentage !== undefined ? `${percentage}%` : <span></span>}{value !== undefined ? value : ""}
-            </div>
+        { isLoading ? (
+          <SkeletonWrapper height={79} count={2} />
```

Comment:
Make count 1 or remove entirely, then increase height to match card

Reply:
I intended to make it 2. Looked a lot nicer to me